site/src/components/IconField/EmojiPicker.tsx

```diff
@@ -26,6 +31,22 @@ type EmojiPickerProps = Omit<
 >;
 
 const EmojiPicker: FC<EmojiPickerProps> = (props) => {
+	/**
+	 * Workaround for a bug in the emoji-mart library where custom emoji images render improperly.
+	 * Setting the image width to 100% ensures they display correctly.
+	 *
+	 * Reference: https://github.com/missive/emoji-mart/pull/806
```

Comment:
Perhaps link the original issue too: https://github.com/missive/emoji-mart/issues/805